identifiers/country-fr/constituencies.csv

```diff
@@ -0,0 +1,578 @@
+id,name
+ocd-division/country:fr/region:ar/department:ai/ed:1,Première circonscription de l'Ain
```

Comment:
To keep in-line with the recent discussions, we should remove the extra nesting. So this OCD ID would become something like
ocd-division/country:fr/ed:ar-ai-1

What do you think?

Reply:
Sorry for the long break, but you know ;) UK Elections

Actually a good idea. But of course it could be that we might run into a problem at some point, should there be 'statelists', like in Austria, Germany or Hungary, where there are national, state and constituency lists.

So I would suggest to leave them in if the construct is not fundamentally wrong?
What do you think?

Reply:
I'm not sure what you mean by "statelists", could you elaborate?

If you're talking about where different level of governments have different sets of consituencies I think we would have something like:

Country level (taking another look, I think department abbreviations are unique across all departments): ocd-division/country:fr/ed:ai-1
region-level: ocd-division/country:fr/region:ar/red:ai-1
department-level ocd-division/coutnry:fr/region:ar/district:ai/sed:1

Thoughts?